src/components/AppProvider/utilities/ScrollLockManager/ScrollLockManager.ts

```diff
@@ -1,6 +1,5 @@
 import {isServer} from '../../../../utilities/target';
 
-export const SCROLL_LOCKING_ATTRIBUTE = 'data-lock-scrolling';
```

Comment:
Looks like this is still being exported in a couple of places. Maybe that's the reason for your failing test? Interestingly I wanted to tophat on web and building consumer failed, which is why I caught this, but my playground ran fine.

Reply:
I suppose this is a breaking change, then! I'll make 2 PRs, as the modal refactor should go in v4, but the change to `html` can go in v3.

Reply:
We used this internally in a few places but it doesn't look like it was ever exported publically. This doesn't need to go to v4 because of this

Reply:
Oh right! I'll edit the changelog and remove the item that mentions this constant. 

However, I now consider this is a breaking change from a CSS point of view, as anything that performs full-page scroll locking (outside of Polaris) should stop doing so on the body and apply it to the html element instead (or they'll have double scrollbars). I'll mention that in the changelog as well. 

If you disagree, I'm happy to consider rebasing onto master again.

Reply:
The CSS behaviour change does feel a bit scary, it's one of those /should/ be fine things but as we're on the cusp of V4 we might as well be safe :) +1 to keeping this in v4

Reply:
You could also not change the name of the constant. We simply moved the attribute from the body to the first child, it was a wrapper before as well.

Reply:
This is a great idea!

I went ahead and renamed the data attribute as well as the name of the constant:

https://github.com/Shopify/polaris-react/pull/1854/commits/cdb175bf565099312ab13b092c405168c0a0d725